card_test.go

```diff
@@ -5,8 +5,39 @@ import (
 	"testing"
 
 	assert "github.com/stretchr/testify/require"
+	"github.com/stripe/stripe-go/form"
 )
 
+func TestCardListParams_AppendTo(t *testing.T) {
+	// Adds `object` for account (this will hit the external accounts endpoint)
+	{
+		params := &CardListParams{Account: String("acct_123")}
+		body := &form.Values{}
+		form.AppendTo(body, params)
+		t.Logf("body = %+v", body)
+		assert.Equal(t, []string{"card"}, body.Get("object"))
+	}
+
+	// Adds `object` for customer (this will hit the sources endpoint)
+	{
+		params := &CardListParams{Account: String("acct_123")}
```

Comment:
This should be `Customer: String("cus_123")` instead

Reply:
Oops! Thank you. Fixed.